docs/how-to-deploy-cp4waiops-customsize.md

```diff
@@ -0,0 +1,353 @@
+| cp4waiops.aiManager.enabled           | bool   | true               | Specify whether or not to install AI Manager.
+| cp4waiops.aiManager.namespace         | string | cp4waiops          | The namespace where AI Manager is installed.
+| cp4waiops.aiManager.instanceName      | string | aiops-installation | The instance name of AI Manager.
+| cp4waiops.eventManager.enabled        | bool   | true               | Specify whether or not to install Event Manager.
+| cp4waiops.eventManager.namespace      | string | noi                | The namespace where Event Manager is installed.
+| cp4waiops.eventManager.clusterDomain  | string | REPLACE_IT         | The domain name of the cluster where Event Manager is installed.
+
+NOTE:
+
+- For `cp4waiops.profile`, the profile `x-small` is only for demo, PoC, or dev environment. If you are looking for official installation, use profile such as `small` or `large` instead.
+- For `cp4waiops.eventManager.enabled`, it needs to be false if you use `x-small` profile as it only covers AI Manager, not including Event Manager.
+- For `cp4waiops.eventManager.clusterDomain`, it is the domain name of the cluster where Event Manager is installed. Use fully qualified domain name (FQDN), e.g.: apps.clustername.abc.xyz.com.
+
+Besides that, the all-in-one configuration exposes a few more install parameters invisible from UI that allows you to customize the install behavior in a more fine-grained manner. Below is just a list of some available parameters. To learn more on the usage of such parameters, please refer to [CP4WAIOps Customized Install Options Using GitOps](./cp4waiops-custom-install.md).
+
+| Parameter                             | Type   | Default Value | Description 
+| ------------------------------------- |--------|---------------|-----------------------------------
+| cp4waiops.storageClass                | string | rook-cephfs   | The storage class for CP4WAIOps to use.
+| cp4waiops.storageClassLargeBlock      | string | rook-cephfs   | The storage class for large block for CP4WAIOps to use.
+| globalImagePullSecrets                | array  | n/a           | A list of registries for image pull when needed during the install.
+
+For example, if the custom build to be installed includes images from registries other than the official IBM entitled registry, you can use `globalImagePullSecrets` to specify all necessary information for these registries including registry URLs, as well as username and password to access these registries.
+
+Again, since these parameters are invisible, you can add them when filling in the form in `HELM` > `VALUES` field:
+
+```yaml
+globalImagePullSecrets:
+- registry: <my_own_registry_1>
+  username: <username_to_registry_1>
+  password: <password_to_registry_1>
+- registry: <my_own_registry_2>
+  username: <username_to_registry_2>
+  password: <password_to_registry_2>
+```
+
+
+
+#### Install *x-small* custom sizing CP4WAIOps 
```

Comment:
What do the following sections mean? Readers may be confused w/o more explanation. It looks the `x-small-<variation>` params have already been mentioned in previous section.

Reply:
Agree to remove all those sections.